src/commonMain/kotlin/microBenchmarks/StringBenchmark.kt

```diff
@@ -120,4 +124,23 @@ class StringBenchmark {
         }
         return sum
     }
+
+    @Benchmark
+    fun stringInterpolation(): Int {
```

Comment:
@alex28sh Might be worth having similar tests with explicit concatenation and explicit StringBuilder usage.